AppVuelos/screens/login.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {SafeAreaView, View, Text} from 'react-native';
+import Input from '../components/Input';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+import {Link} from '@react-navigation/native';
+import Buttons from '../components/buttons';
+import loginStyles from '../styles/LoginStyles';
+
+function Login(): JSX.Element {
+  return (
+    <SafeAreaView style={loginStyles.container}>
+      <View style={loginStyles.rowContainer}>
+        <Text style={loginStyles.title}>Flight Bookings</Text>
+        <FontAwesome name="plane" size={30} color={'#5C6EF8'} />
```

Comment:
Probably you should use primaryColor constant instead of hex value directly

Reply:
Done